### PR TITLE
soap for maints drones!

### PR DESCRIPTION
**Pull request by deathrobotpunch**
## About The Pull Request

adds soap to maint drones' internal storage
## Why It's Good For The Game
eliminates the hell that is no soap spawning on station meaning you are unable to clean at all. also feel free to recommend more internal storage items too!
## Changelog
:cl:
qol: drones now have soap in their internal storage!
/:cl:

### CODE REVIEW

**Comment by norsvenska** [comment]
what happens when the soap runs out

**Comment by Loafin34** [comment]
inf soap in chemistry

**Comment by deathrobotpunch** [comment]
> what happens when the soap runs out
> 
> also you should add it to the populatecontents

I didn’t think of that lol, I’ll update it

**Comment by deathrobotpunch** [comment]
Ok I gave them the cyborg soap module which shouldn’t run out and added it to populate contents

**Comment by deathrobotpunch** [comment]
turns out I did an oopsie and made a blank populate item var

**Comment by deathrobotpunch** [comment]
turns out that cyborg soap had a limited amount of uses and had to be restocked by a charger. so I created a soap for drones specifically.

this version has a stupidly large amount of uses so you shouldn’t run out of it.

though I changed the wash speed to be on par with syndie soap (0.5 seconds), if any maints have an issue with it I’m willing to change it.

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -89,6 +89,13 @@
 	worn_icon_state = "soapsyndie"
 	cleanspeed = 0.5 SECONDS //faster than mops so it's useful for traitors who want to clean crime scenes
 
+/obj/item/soap/drone
+	name = "\improper integrated soap module"
+	inhand_icon_state = "soapnt"
+	worn_icon_state = "soapnt"
+	cleanspeed = 0.5 SECONDS //can be changed if someone isn't happy
+	uses = 999999999 //turns out cyborg soap has limited uses so I summon stupidly large number
```
Comment (Time-Green) [inline code/game/objects/items/clown_items.dm:97]:
```suggestion
	uses = INFINITY
```

File: code/modules/mob/living/basic/drone/drone_tools.dm [hunk 2]
```diff
@@ -38,8 +39,7 @@
 	builtintools += new /obj/item/multitool/drone(src)
 	builtintools += new /obj/item/pipe_dispenser(src)
 	builtintools += new /obj/item/t_scanner(src)
-	builtintools += new /obj/item/analyzer(src)
```
Comment (Time-Green) [inline code/modules/mob/living/basic/drone/drone_tools.dm:41]: Analyzer removal intended?

**Review by Time-Green — CHANGES_REQUESTED** [review]
Small remarks. Also dronemaid powercreep strikes again

File: code/modules/mob/living/basic/drone/drone_tools.dm (hunk 2 above)
Comment (deathrobotpunch) [reply code/modules/mob/living/basic/drone/drone_tools.dm:41]: nope, readded it

**Comment by deathrobotpunch** [comment]
> Small remarks. Also dronemaid powercreep strikes again

changes made!

**Pull request merged**